src/reimer_arenkiel_lab_to_nwb/dj_utils.py

```diff
@@ -97,15 +93,13 @@ def add_subject(nwbfile: NWBFile, key: dict = None, verbose: bool = False) -> No
         date_of_birth=datetime.datetime.combine(subject_info['dob'], datetime.datetime.min.time()),
         sex=subject_info["sex"] if subject_info["sex"] in ("M", "F") else "U",
         description=subject_info["mouse_notes"] or "no notes",
-        species="Mus musculus",
+        species="Mus Musculus",
```

Comment:
why make the second part uppercase? has there been a change in best practices?

Reply:
yeah this should be "Mus musculus" as before (see inspector snippet below)

Reply:
Actually don't know why I changed it

Reply:
I changed it back to the correct case